test/integration/query.spec.js

```diff
@@ -577,5 +577,25 @@ describe('Query operations', () => {
       response.extent.spatialReference.should.have.property('wkid', 3857)
       response.should.not.have.property('count')
     })
+
+    it('should return extent when predefined', () => {
+      const response = FeatureServer.query({
```

Comment:
should returnCountOnly=true be part of this test?

Reply:
No, but I should probably add a test to cover that case.